neonvm/tools/vm-builder/main.go

```diff
@@ -250,24 +251,22 @@ fi
 ::respawn:su -p nobody -c '/usr/local/bin/pgbouncer /etc/pgbouncer.ini'
 ::respawn:su -p nobody -c 'DATA_SOURCE_NAME="user=cloud_admin sslmode=disable dbname=postgres" /bin/postgres_exporter --auto-discover-databases --exclude-databases=template0,template1'
 ttyS0::respawn:/neonvm/bin/agetty --8bits --local-line --noissue --noclear --noreset --host console --login-program /neonvm/bin/login --login-pause --autologin root 115200 ttyS0 linux
+::shutdown:/neonvm/bin/vmshutdown
 `
 
 	scriptVmAcpi = `
 event=button/power
-action=/neonvm/bin/powerdown
+action=/neonvm/bin/poweroff
 `
 
-	scriptPowerDown = `#!/neonvm/bin/sh
-
+	scriptVmShutdown = `#!/neonvm/bin/sh
+rm /neonvm/vmstart.allowed
+if [ -e /neonvm/vmstart.allowed ]; then
+	echo "Error: could not remove vmstart.allowed marker, might hang indefinitely during shutdown" >2&
```

Comment:
```suggestion
	echo "Error: could not remove vmstart.allowed marker, might hang indefinitely during shutdown" >&2
```
assuming this was meant to go to stderr, right? (rather than running "write to the file '2' in the background")

Reply:
Yeah, noticed independently, but even with the fix 03ae60de02e49137bc35bd1eea23e669f97110c9 , the log messages don't show up in staging logs

Reply:
Gotta correct myself, it works: https://neonprod.grafana.net/explore?orgId=1&left=%7B%22datasource%22:%22xHHYY0dVz%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22xHHYY0dVz%22%7D,%22editorMode%22:%22code%22,%22expr%22:%22%7Bneon_region%3D%5C%22us-east-2%5C%22%7D%20%7C%3D%20%60compute-royal-firefly-37505771%60%20%7C%3D%20%60workload%20shut%20down%60%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%221695580354571%22,%22to%22:%221695580426866%22%7D%7D

Reply:
Going to retry a few times to make sure